diff -up kwallet-pam-5.16.1/pam_kwallet_init.1~ kwallet-pam-5.16.1/pam_kwallet_init
--- kwallet-pam-5.16.1/pam_kwallet_init.1~	2019-06-20 21:59:36.699700401 +0200
+++ kwallet-pam-5.16.1/pam_kwallet_init	2019-06-20 22:01:11.333536667 +0200
@@ -1,6 +1,6 @@
 #!/bin/sh
 
 if test -n "$PAM_KWALLET5_LOGIN" ; then
-    env | socat STDIN UNIX-CONNECT:$PAM_KWALLET5_LOGIN
+    env | socat -u STDIN UNIX-CONNECT:$PAM_KWALLET5_LOGIN
 fi
 
